extensions/container-image/deployment/src/main/java/io/quarkus/container/image/deployment/ContainerImageConfig.java

```diff
@@ -26,6 +27,12 @@
     @ConfigItem(defaultValue = "${quarkus.application.version:latest}")
     public Optional<String> tag;
 
+    /**
+     * Additional tags of the container image.
+     */
+    @ConfigItem
+    public Optional<Set<String>> additionalTags;
```

Comment:
I would prefer if we used `List` instead of `Set` here. That would make it consistent with the rest of the codebase